python/cuml/cuml_accel_tests/test_basic_estimators.py

```diff
@@ -136,97 +134,3 @@ def test_k_neighbors_regressor():
         for metric in ["euclidean", "manhattan"]:
             knr = KNeighborsRegressor().fit(X, y)
             knr.predict(X)
-
```

Comment:
Moved generic tests for the estimator proxy into `test_estimator_proxy.py`. I _think_ in the long run the tests above should be moved into `integration` as tests for specific estimators and this whole file can be deleted. Did the minimal change for now.

Reply:
Fully agree with that change. This file existed more because it was the first test file I wrote as just a few estimators were there, not that we have more and the design is revamped, that proposal to eliminate this file makes total sense.